tests/data/cms_mangrove_canopy/data.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import hashlib
+import os
+import random
+import shutil
+
+import numpy as np
+import rasterio
+
+np.random.seed(0)
+random.seed(0)
+
+SIZE = 64
+
+
+files = [
+    {"image": "Mangrove_agb_Angola.tif"},
+    {"image": "Mangrove_hba95_Angola.tif"},
+    {"image": "Mangrove_hmax95_Angola.tif"},
+]
+
+
+def create_file(path: str, dtype: str, num_channels: int) -> None:
+    profile = {}
+    profile["driver"] = "GTiff"
+    profile["dtype"] = dtype
+    profile["count"] = num_channels
+    profile["crs"] = "epsg:4326"
+    profile["transform"] = rasterio.transform.from_bounds(0, 0, 1, 1, 1, 1)
+    profile["height"] = SIZE
+    profile["width"] = SIZE
```

Comment:
could add:
```
profile["compress"] = "lzw"
profile["predictor"] = 2
```
for compression (smaller test files).

See https://gdal.org/drivers/raster/gtiff.html#creation-options for things you can put here.